services-manager/src/main/resources/application-discovery.yaml

```diff
@@ -27,5 +27,7 @@ manager:
   services:
     discovery-service:
       jar: discovery-service/target/discovery-service-${app.version}-exec.jar
+      profiles:
+        - discovery
```

Comment:
Why start the discovery service in discovery profile?

Reply:
Not sure why I added that. It has now been removed.